tox.ini

```diff
@@ -1,29 +1,21 @@
 [tox]
-envlist = python
 minversion = 3.2
+envlist = py36
+skipsdist = true
 requires =
 	tox-venv
 
 [testenv]
-commands =
-	python -m unittest discover
 extras =
 	dev
 usedevelop = True
+commands =
+    check-manifest
+	python -m unittest discover
```

Comment:
It's a bit weird to see two commands with such differing indentations.

(Are you using literal tabs by any chance, in an editor where tab size is not set to 8?)

Reply:
Ah, sorry about that. I was also in the middle of reconfiguring my editor at the time.

Reply:
854f8f85318e98eb86799ed3d2091cc456261d17